tests/sentry/sentry_metrics/test_indexer.py

```diff
@@ -5,7 +5,7 @@
 
 def test_resolve():
     assert INDEXER.resolve(1, "what") is None
-    assert INDEXER.resolve(1, "sentry.sessions.user") == 11
+    assert INDEXER.resolve(0, "sentry.sessions.user") == 11
```

Comment:
Shouldn't `INDEXER.resolve(1, "sentry.sessions.user")` also work? We could have both assertions here.